db/seeds.rb

```diff
@@ -0,0 +1,13 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+
+greetingsExample = ["Hello", "Hola", "Bonjour", "Ciao", "Hallo", "Hej", "Ahoj", "Konnichiwa", "Anyoung", "Salam", "Merhaba", "Szia", "Namaste", "Sawubona"]
+
+greetingsExample.each do |greeting|
+    Greeting.create(text: greeting)
```

Comment:
- [ ] [OPTIONAL] It would be great to have a step in your README for users to add the data in your seed file into the database. Something like 

```
rails db:seed
```

Beyond this, you did excellently well on this project. welldone

Reply:
Oh I forgot to do it thank you